### PR TITLE
Add ORCA backend

### PR DESCRIPTION
**Pull request by Omar-ORCA**
### Description
Implementation of the ORCA remote REST QPU

### CODE REVIEW

File: scripts/validate_container.sh [hunk 1]
```diff
@@ -48,12 +48,12 @@ available_backends=`\
     do
         libEM=$(cat $file | grep "LIBRARY_MODE_EXECUTION_MANAGER=")
         if grep -q "LIBRARY_MODE_EXECUTION_MANAGER=" $file ; then 
-          continue
+        continue
```
Comment (bmhowe23) [inline scripts/validate_container.sh:51]: Suggested change to restore spacing back to what it was before the interim changes.
```suggestion
          continue
```

**Review by bmhowe23 — APPROVED** [review]
Thanks for your contribution!

**Comment by bettinaheim** [comment]
Marking this as 0.6 so that we don't forget to highlight. The implementation is included in 0.5, but there are no docs/ways to request access at this time.

**Pull request merged**